src/components/SearchAndSort/SearchAndSort.js

```diff
@@ -753,19 +718,25 @@ export class SearchAndSort extends Component {
       />
     );
 
+    const header = renderProps => (
```

Comment:
Move this function to the class methods and rename it to "renderPaneHeader" for example, please.

Reply:
Done!